packages/frontend/src/creation-form/index.tsx

```diff
@@ -277,28 +277,23 @@ export const Component = ({
             <div className="flex flex-col px-4 gap-4">
                 <Typography variant="h3">Goal</Typography>
                 <div className="flex flex-col gap-2 md:flex-row">
-                    <div className="w-full">
-                        <Select
-                            id="constraint-type"
-                            className={{
-                                root: "w-full",
-                                input: "w-full",
-                                inputWrapper: "w-full",
-                            }}
-                            label={t("label.constraint.type")}
-                            messages={{
-                                noResults: t("select.no.results"),
-                            }}
-                            info={
-                                <Typography variant="sm">
-                                    {t("info.constraint.type")}
-                                </Typography>
-                            }
-                            placeholder={t("placeholder.pick.constraint.type")}
-                            onChange={setConstraintType}
-                            options={CONSTRAINT_TYPES}
-                            value={constraintType || null}
-                        />
+                    <div className="flex w-full flex-wrap gap-3">
+                        {CONSTRAINT_TYPES.map((ct) => {
+                            return (
+                                <Chip
+                                    key={ct.value}
+                                    size="big"
+                                    clickable
+                                    active={constraintType?.value === ct.value}
+                                    onClick={() => {
+                                        console.log("click", ct);
```

Comment:
Left a `console.log` here.

Reply:
Addressed in f31afc1b6f75319915be567a8b8cbfe6805e6a55